servers/navigation_server_3d.h

```diff
@@ -274,9 +274,15 @@ class NavigationServer3D : public Object {
 
 	virtual int get_process_info(ProcessInfo p_info) const = 0;
 
-#ifdef DEBUG_ENABLED
+public:
+	void set_debug_enabled(bool p_enabled);
+	bool get_debug_enabled() const;
+
 private:
 	bool debug_enabled = false;
+
+#ifdef DEBUG_ENABLED
+private:
```

Comment:
Nit: This `private` scope isn't really doing anything here.

Reply:
ooooh the rebase chaos.